lib/invoke.js

```diff
@@ -112,7 +112,7 @@ function invoke(func, data, funcsDesc, options) {
         resolve(response);
       }
     };
-    core.ns.services.get((err, servicesInfo) => {
+    core.services.get((err, servicesInfo) => {
```

Comment:
this should be `core.ns` since that's the configured namespace. The issue is in line 83. `connectionOptions` is not taking the `namespace` into account. To fix it replace:

```patch
-      const connectionOptions = helpers.getConnectionOptions(helpers.loadKubeConfig());
+      const connectionOptions = helpers.getConnectionOptions(helpers.loadKubeConfig(), {
+        namespace,
+      });
```

Reply:
Yes, tested this solution and it's working well. I've updated the PR as well